GitHubTeams.ps1

```diff
@@ -452,62 +804,648 @@ function New-GitHubTeam
+
+        Removes a project specified by its ID from team 'Everybody' in organization 'PowerShell'.
+
+    .EXAMPLE
+        Remove-GitHubTeamProject `
+            -OrganizationName PowerShell `
+            -TeamName Everybody `
+            -Project 4387531 `
+            -Force
+
+        Removes a project specified by its ID from team 'Everybody' in organization 'PowerShell'
+        without prompting.
+#>
+    [CmdletBinding(
+        SupportsShouldProcess,
+        PositionalBinding = $false,
+        ConfirmImpact = 'High',
+        DefaultParameterSetName = 'TeamSlug')]
+    [Alias('Delete-GitHubTeamRepository')]
+    param
+    (
+        [Parameter(
+            Mandatory,
+            ValueFromPipelineByPropertyName)]
+        [ValidateNotNullOrEmpty()]
+        [String] $OrganizationName,
+
+        [Parameter(
+            Mandatory,
+            ValueFromPipelineByPropertyName,
+            ParameterSetName='TeamName')]
+        [ValidateNotNullOrEmpty()]
+        [String] $TeamName,
+
+        [Parameter(
+            Mandatory,
+            ValueFromPipelineByPropertyName,
+            ParameterSetName='TeamSlug')]
+        [string] $TeamSlug,
+
+        [Parameter(
+            Mandatory,
+            ValueFromPipelineByPropertyName)]
+        [Alias('ProjectId')]
+        [int64] $Project,
+
+        [switch] $Force,
+
+        [string] $AccessToken
+    )
+
+    Write-InvocationLog
+
+    $telemetryProperties = @{
+        OrganizationName = (Get-PiiSafeString -PlainText $OrganizationName)
+        TeamName = (Get-PiiSafeString -PlainText $TeamName)
+        TeamSlug = (Get-PiiSafeString -PlainText $TeamSlug)
+        Project = (Get-PiiSafeString -PlainText $Project.ToString([CultureInfo]::InvariantCulture))
+    }
+
+    if ($PSCmdlet.ParameterSetName -eq 'TeamName')
+    {
+        $getGitHubTeamParms = @{
+            OrganizationName = $OrganizationName
+            TeamName = $TeamName
+        }
+
+        if ($PSBoundParameters.ContainsKey('AccessToken'))
+        {
+            $getGitHubTeamParms['AccessToken'] = $AccessToken
+        }
+
+        $team = Get-GitHubTeam @getGitHubTeamParms
+        $TeamSlug = $team.slug
+    }
+
+    if ($Force -and (-not $Confirm))
+    {
+        $ConfirmPreference = 'None'
+    }
+
+    if (-not $PSCmdlet.ShouldProcess($TeamName, 'Remove Github Team Project'))
+    {
+        return
+    }
+
+    $params = @{
+        UriFragment =  "/orgs/$OrganizationName/teams/$TeamSlug/projects/$Project"
+        Method = 'Delete'
+        Description =  "Removing project $Project from team $TeamSlug"
+        AccessToken = $AccessToken
+        TelemetryEventName = $MyInvocation.MyCommand.Name
+        TelemetryProperties = $telemetryProperties
+        AcceptHeader = $script:inertiaAcceptHeader
+    }
+
+    Invoke-GHRestMethod @params | Out-Null
+}
+
+filter Remove-GitHubTeamRepository
+{
+<#
+    .SYNOPSIS
+        Removes a team repository from an organization on GitHub.
```

Comment:
Same issue here as above, both in the name of the command as well as the description.  I don't yet have an alternate suggestion on the name.

```suggestion
Removes an organization repository from a team on GitHub.  This does not delete the repository, it just removes it from the team.
```